src/app/external-cluster-wizard/steps/external-cluster/provider/eks/template.html

```diff
@@ -73,38 +73,40 @@
       </mat-form-field>
     </ng-template>
 
-    <mat-form-field>
-      <mat-label>{{isDialogView() ? "Node IAM role" : "Cluster Service Role"}}</mat-label>
+    <mat-form-field *ngIf="isDialogView()">
+      <mat-label>Node IAM role</mat-label>
       <input matInput
              type="text"
              autocomplete="off"
              [formControlName]="Controls.RoleArn" />
-
-      <mat-hint *ngIf="!isDialogView(); else dialogView">
-        Enter the IAM role arn to allow the Kubernetes control plane to manage AWS resources on your behalf.
-        <a href="https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html"
+      <mat-hint>
+        Enter the IAM role arn that will be used by the nodes.
+        <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-node-role.html"
            target="_blank"
            rel="noreferrer noopener">
           EKS user guide >
         </a>
       </mat-hint>
 
-      <ng-template #dialogView>
-        <mat-hint>
-          Enter the IAM role arn that will be used by the nodes.
-          <a href="https://docs.aws.amazon.com/eks/latest/userguide/create-node-role.html"
-             target="_blank"
-             rel="noreferrer noopener">
-            EKS user guide >
-          </a>
-        </mat-hint>
-      </ng-template>
-
       <mat-error *ngIf="form.get(Controls.RoleArn).hasError('required')">
         Role is <strong>required</strong>.
       </mat-error>
     </mat-form-field>
 
+    <km-combobox *ngIf="!isDialogView()"
+                 filterBy="roleName"
+                 inputLabel="Select Role... "
+                 [label]="roleArnLabel"
+                 hint="Select the IAM role arn to allow the Kubernetes control plane to manage AWS resources on your behalf."
```

Comment:
move this above [label] so that static values should be above as most of the convention is followed + readability

Reply:
there  is a link (`<a>` tag )in the previous hint do we need it ??WDYT?

Reply:
ah! good point we can't handle link in combobox as we can't pass html tags and rendered as innerHtml in component i think for this we need to add similar to what autocomplete component.

cc: @Waseem826

Reply:
Do we need this or not i am not sure but @imharshita can answer if it is important for user we need to cater this than.

Reply:
i think i can add it using`<mat-hint>` after the `<km-combobox>` and using some style for it if its required